apps/src/music/MusicView.jsx

```diff
@@ -122,14 +121,12 @@ class UnconnectedMusicView extends React.Component {
 
     document.body.addEventListener('keyup', this.handleKeyUp);
 
-    this.initBlockly();
-
-    setInterval(this.updateTimer, 1000 / 30);
-
     this.loadLibrary().then(library => {
       this.setState({library});
+      this.initBlockly();
```

Comment:
Interesting, I remember trying to move blockly loading after the manifest was loaded, similar to this, and it seemed to break blockly. I wonder why this is working now...
Edit: oh maybe it was just because i didn't catch the null workspace case in `resizeBlockly()` 🙂